### PR TITLE
Add log-in component styles

### PR DESCRIPTION
**Pull request by Ponomaleks**

### CODE REVIEW

File: src/app/auth/pages/log-in/log-in.component.html [hunk 1]
```diff
@@ -1,5 +1,15 @@
-<a [routerLink]="['/', appRoutes.Auth, appRoutes.RestorePassword]">Forgot password?</a>
-<form>
-  <button type="button" (click)="onLogIn()">Log In</button>
+<img class="logo" src="assets/icons/logo_dark.svg" alt="logo" />
```
Comment (GennadiyShpak) [inline src/app/auth/pages/log-in/log-in.component.html:1]: where widht and height attribute?

File: src/app/auth/pages/log-in/log-in.component.html [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+<h2 class="title">U-pamers</h2>
+<h3 class="subtitle">Log in to receive full experience and start your journey with us</h3>
+<form class="form">
+  <epm-input class="password"></epm-input>
+  <epm-input class="confirm"></epm-input>
```
Comment (GennadiyShpak) [inline src/app/auth/pages/log-in/log-in.component.html:6]: I think we dont need closed part of this tag

File: src/app/auth/pages/log-in/log-in.component.ts [hunk 3]
```diff
@@ -1,17 +1,21 @@
...
   selector: 'epm-log-in',
   standalone: true,
-  imports: [CommonModule, RouterLink],
+  imports: [CommonModule, RouterLink, EpmInputComponent, EpmButtonComponent],
   templateUrl: './log-in.component.html',
   styleUrls: ['./log-in.component.scss']
 })
 export class LogInComponent {
   readonly appRoutes: typeof APP_ROUTER_NAME = APP_ROUTER_NAME;
+  protected readonly BUTTON_THEMES = BUTTON_THEMES;
```
Comment (GennadiyShpak) [inline src/app/auth/pages/log-in/log-in.component.ts:18]: where is type? why capital case? do we use protected propperty in our application?

Comment (Ponomaleks) [reply src/app/auth/pages/log-in/log-in.component.ts]: Changed

File: src/app/auth/pages/log-in/log-in.component.html (hunk 2 above)
Comment (Ponomaleks) [reply src/app/auth/pages/log-in/log-in.component.html]: removed closing tag

File: src/app/auth/pages/log-in/log-in.component.html (hunk 1 above)
Comment (Ponomaleks) [reply src/app/auth/pages/log-in/log-in.component.html]: added

**Review by GennadiyShpak — APPROVED** [review]

**Pull request merged**